packages/amplication-server/src/core/entity/dto/UpdateOneEntityFieldArgs.ts

```diff
@@ -15,4 +15,7 @@ export class UpdateOneEntityFieldArgs {
 
   @Field(() => String, { nullable: true })
   relatedFieldDisplayName?: string;
+
+  @Field(() => Boolean, { nullable: true })
+  relatedFieldAllowMultipleSelection?: boolean;
```

Comment:
are we using this? can we really update the related field when updating the the other side of the relation?

Reply:
Unless I'm missing something, the answer is the same as above

![image](https://github.com/amplication/amplication/assets/39680385/a1475184-51f3-4073-80e0-7df6b468dde4)